### DISCUSSION TITLE
Scheduling and automating a wol startup

### DISCUSSION THREAD

**Discussion by Abygail007** [amidaware/tacticalrmm#1704] [Q&A]
Hello everybody ! sorry for my poor English !

I know that tacticalRMM now includes WakeOnLan. I use automation a lot via Trmm but some machines are sometimes turned off which is not practical.

Do you have a solution for automating / planning a WoL launcher via Tactical rmm?

thank you very much!

**Comment by silversword411** [comment]
This made me think about adding that to the Windows Update Rework so I did: https://github.com/amidaware/tacticalrmm/issues/1188

- Attempt WoL before running updates

There is a WoL script in the Community scripts, but I think it needs some work: https://github.com/amidaware/community-scripts/blob/5890cf8d7ed6377280cc685aa145b7c2b3546024/scripts/Win_WOL_Test_State.ps1#L43

**Comment by bbrendon** [comment]
You can use the API to automate WoL

**Reply by Abygail007** [reply]
Hello, thank you very much for the help. Unfortunately, I'm not very familiar with the API. I've been trying to find out about it since this morning, but I don't really understand what to do.

**Reply by plao** [reply]
Hi @bbrendon , I am also interested on a similar task (global WOL on all net devices and scheduled execution of some tasks...). Did you could please provide me/us more infos? Thanks!
